File: matlab/src/solvers/SSA.m
classdef SSA < SolverSSA
    % SSA - Alias for SolverSSA (Stochastic State-space Analysis solver)
    %
    % Copyright (c) 2012-2026, Imperial College London
    % All rights reserved.

    methods
        function self = SSA(model, varargin)
            % SSA(MODEL, VARARGIN)
            self@SolverSSA(model, varargin{:});
        end
    end
end

File: jar/src/main/java/jline/solvers/ssa/SSA.java
/*
 * Copyright (c) 2012-2026, QORE Lab, Imperial College London
 * All rights reserved.
 */

package jline.solvers.ssa;

import jline.lang.Network;
import jline.solvers.SolverOptions;

/**
 * SSA is an alias for SolverSSA (Stochastic State-space Analysis solver).
 */
public class SSA extends SolverSSA {

    public SSA(Network model, String method) {
        super(model, method);
    }

    public SSA(Network model, SolverOptions options) {
        super(model, options);
    }

    public SSA(Network model, Object... varargin) {
        super(model, varargin);
    }

    public SSA(Network model) {
        super(model);
    }
}
